pkg/image/admission/imagestream_limits.go

```diff
@@ -17,79 +12,84 @@ type LimitVerifier interface {
 	VerifyLimits(namespace string, is *imageapi.ImageStream) error
 }
 
-func NewLimitVerifier(client kclient.LimitRangesNamespacer) LimitVerifier {
-	lw := &cache.ListWatch{
-		ListFunc: func(options kapi.ListOptions) (runtime.Object, error) {
-			return client.LimitRanges(kapi.NamespaceAll).List(options)
-		},
-		WatchFunc: func(options kapi.ListOptions) (watch.Interface, error) {
-			return client.LimitRanges(kapi.NamespaceAll).Watch(options)
-		},
-	}
-	indexer, reflector := cache.NewNamespaceKeyedIndexerAndReflector(lw, &kapi.LimitRange{}, 0)
-	reflector.Run()
+type UninitializedLimitVerifier interface {
```

Comment:
Dead code now, will remove

On Sep 14, 2016, at 1:43 PM, David Eads <notifications@github.com> wrote:

In pkg/image/admission/imagestream_limits.go
<https://github.com/openshift/origin/pull/10777#discussion_r78798076>:

> @@ -17,79 +12,84 @@ type LimitVerifier interface {
>  	VerifyLimits(namespace string, is *imageapi.ImageStream) error
>  }
>
> -func NewLimitVerifier(client kclient.LimitRangesNamespacer) LimitVerifier {
> -	lw := &cache.ListWatch{
> -		ListFunc: func(options kapi.ListOptions) (runtime.Object, error) {
> -			return client.LimitRanges(kapi.NamespaceAll).List(options)
> -		},
> -		WatchFunc: func(options kapi.ListOptions) (watch.Interface, error) {
> -			return client.LimitRanges(kapi.NamespaceAll).Watch(options)
> -		},
> -	}
> -	indexer, reflector := cache.NewNamespaceKeyedIndexerAndReflector(lw, &kapi.LimitRange{}, 0)
> -	reflector.Run()
> +type UninitializedLimitVerifier interface {

Where is this used?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/10777/files/12470c9c4ca8ef7c2e76b882442b035f3e1471dd..339b33209a5566822f43f5db3900287735e055f7#r78798076>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ABG_p3h1IGBA2_2LCDQ6H_VqGkYrAwzcks5qqDJDgaJpZM4JzJrj>
.